### DISCUSSION TITLE
plugins = numpy.typing.mypy_plugin

### DISCUSSION THREAD

**Discussion by rootsmusic** [microsoft/vscode-mypy#240] [Q&A] [answered]
My environment has mypy 1.8.0 and mypy_extensions 1.0.0.  I'd like to enable the [mypy plugin for numpy](https://numpy.org/doc/stable/reference/typing.html#mypy-plugin).  What's the path for this extension's [mypy.ini](https://mypy.readthedocs.io/en/stable/config_file.html) (I see it in many directories)?  Thanks.

**Comment by karthiknadig** [comment, marked as answer]
If you have the mypy plugin from numpy installed in your environment it should work automatically. The extension doesn't select `mypy.ini` , we let mypy decide which one it wants to use.
